dxr/build.py

```diff
@@ -523,6 +522,11 @@ def docs():
                     total['annotations'] = annotations_for_this_line
                 yield es.index_op(total)
 
+                # Because needles_by_line holds a reference, total is not
```

Comment:
Can you point out where needles_by_line takes on a reference to `total`? I'm not seeing it.